app/api/dao/admin.py

```diff
@@ -60,6 +69,15 @@ def revoke_admin_user(user_id, data):
 
         new_admin_user = UserModel.find_by_id(admin_user_id)
 
+        #For test purposes
+        admin_user = UserModel.find_by_id(user_id)
+
+        if admin_user: 
+            if not admin_user.is_admin:
+                return messages.USER_ASSIGN_NOT_ADMIN, 403
```

Comment:
```suggestion
                return messages.USER_REVOKE_NOT_ADMIN, 403
```

the message being used should be about revoking a user and not assigning